features/admin.applications.v1/components/settings/sign-on-methods/step-based-flow/add-authenticator-modal.tsx

```diff
@@ -177,16 +177,18 @@ export const AddAuthenticatorModal: FunctionComponent<AddAuthenticatorModalProps
     const { getLink } = useDocumentation();
     const { hiddenAuthenticators } = useAuthenticationFlow();
     const { deploymentConfig } = useDeploymentConfig();
+    const { UIConfig } = useUIConfig();
+    const dispatch: Dispatch = useDispatch();
 
     const isSAASDeployment: boolean = useSelector((state: AppState) => state?.config?.ui?.isSAASDeployment);
 
-    const hiddenConnectionTemplates: string[] = useSelector(
```

Comment:
won't we need this to filter out the connection templates marked as hidden from the deployment config?

Reply:
This is handled in the hook which returns the Connections templates list.